gammapy/time/lightcurve.py

```diff
@@ -274,14 +274,50 @@ def _get_off_evt_list(spec_extract):
     @staticmethod
     def _get_on_evt_list(spec_extract):
         """
-        Returns list of OFF events for each observations
+        Returns list of ON events for each observations
         """
         on_evt_list = []
         for obs in spec_extract.bkg_estimate:
             on_evt_list.append(obs.on_events)
 
         return on_evt_list
 
+    def create_fixed_time_bin_lc(self, time_step, spectral_model, energy_range, spectrum_extraction):
+        """
+        Create time intervals of fixed size and then a light curve using those intervals
+
+        :param time_step: float
```

Comment:
Please use the numpy docstring format for the parameters (see any other function / method docstring in Gammapy as example). With this, it won't render correctly in our docs build.
You can check locally via `make doc && make doc-show`, but you don't have to, the Sphinx build is super slow.